_includes/footer.html

```diff
@@ -0,0 +1,24 @@
+<div class="footer text-center mt-auto">
+  <ul>
+    <div class="mb-2">
+      <a href="https://www.facebook.com/groups/pep.dortmund/" role="button" class="btn btn-lg btn-outline-secondary">
+        <span class="fab fa-facebook"></span>
+      </a>
+      <a href="https://twitter.com/pepdortmund" role="button" class="btn btn-lg btn-outline-secondary">
+        <span class="fab fa-twitter"></span>
```

Comment:
Do we still use that account?

Reply:
Maybe better to link Linkedin: https://www.linkedin.com/company/pep-et-al

Reply:
> Do we still use that account?

Latest post from 2022, so I guess not... Will add the LinkedIn URL instead 👍